docs/pages/reference/mesheryctl/test.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+title: test
+permalink: reference/test/main
+redirect_from: reference/test/main/
+type: reference
+display-title: "false"
+language: en
+command: test
+subcommand: nil
+---
+
+# test
+
+
+
+## Options
+
+<pre class='codeblock-pre'>
+<div class='codeblock'>
+  -h, --help   help for test
```

Comment:
@jinronga was this page auto-generated during the test runs? 👀

Reply:
Yes, it seems so. Sorry, I didn't pay attention.

Reply:
@jinronga please remove this file / page.